examples/chat/public/main.js

```diff
@@ -118,19 +114,8 @@ $(function() {
   // options.fade - If the element should fade-in (default = true)
   // options.prepend - If the element should prepend
   //   all other messages (default = false)
-  const addMessageElement = (el, options) => {
-    var $el = $(el);
-
-    // Setup default options
-    if (!options) {
-      options = {};
-    }
-    if (typeof options.fade === 'undefined') {
-      options.fade = true;
-    }
-    if (typeof options.prepend === 'undefined') {
-      options.prepend = false;
-    }
+  const addMessageElement = (el, options = { fade: true, prepend: false }) => {
```

Comment:
'X is typing' message not fading out.
I tried converting all the arrow function to normal function and it worked. But it's not working in arrow function.

Reply:
👌